roadmap.md

```diff
@@ -0,0 +1,81 @@
+#OpenMetadata Roadmap
```

Comment:
Is this file required? I prefer roadmap.md being in the main folder instead of docs folder. wdyt?